### PR TITLE
Fix primary element color variables

### PR DESCRIPTION
**Pull request by juliusknorr**
In order to have proper variables available to be used with a fallback for very bright colors, this adds the same set of color variables to the theme which based its calculation on the element color.

Required for https://github.com/nextcloud/nextcloud-vue/pull/3070

### CODE REVIEW

**Review by GretaD — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]
:+1: :elephant:

File: apps/theming/lib/Themes/DefaultTheme.php [hunk 1]
```diff
@@ -120,12 +123,15 @@ public function getCSSVariables(): array {
...
-			'--color-primary-element-hover' => $this->util->mix($this->util->elementColor($this->primaryColor), $colorMainBackground, 80),
-			'--color-primary-element-light' => $this->util->lighten($this->util->elementColor($this->primaryColor), 15),
-			'--color-primary-element-lighter' => $this->util->mix($this->util->elementColor($this->primaryColor), $colorMainBackground, -70),
+			'--color-primary-element' => $colorPrimaryElement,
+			'--color-primary-element-text' => $this->util->invertTextColor($colorPrimaryElement) ? '#000000' : '#ffffff',
+			'--color-primary-element-hover' => $this->util->mix($colorPrimaryElement, $colorMainBackground, 60),
+			'--color-primary-element-light' => $colorPrimaryElementLight,
+			'--color-primary-element-light-text' => $colorPrimaryElement,
+			'--color-primary-element-light-hover' => $this->util->mix($colorPrimaryElementLight, $colorMainText, 90),
+			'--color-primary-element-text-dark' => $this->util->darken($this->util->invertTextColor($colorPrimaryElement) ? '#000000' : '#ffffff', 7),
```
Comment (skjnldsv) [inline apps/theming/lib/Themes/DefaultTheme.php:132]: Hey @juliushaertl I see you added `color-primary-element-text-dark` which I cannot find anywhere before.
Was that a mistake? I see only one usage, but I don't think this is actually a necessary variable :)

Comment (juliushaertl) [reply apps/theming/lib/Themes/DefaultTheme.php:132]: I don't remember any use case either. Thinking about it maybe we should also start documenting some context to the variables on when to use?

Comment (skjnldsv) [reply apps/theming/lib/Themes/DefaultTheme.php:132]: That's exactly how I encountered this variable :see_no_evil:

**Pull request merged**